scratch-js/lib/decode-adpcm-audio.mjs

```diff
@@ -0,0 +1,128 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2013-2019 Nathan Dinsmore and Adroitwhiz
```

Comment:
You should probably also credit Truman Kilen, who actually [wrote the decoder](https://github.com/nathan/phosphorus/pull/583).

Reply:
Oh right! Done.